charts/sonarqube/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: sonarqube
 description: SonarQube is an open sourced code quality scanning tool
-version: 9.2.7
+version: 9.2.8
```

Comment:
Please bump the minor version instead as this adds a new feature

Reply:
done